pymc3/variational/advi.py

```diff
@@ -182,7 +182,7 @@ def _calc_elbo(vars, model, n_mcsamples, random_seed):
 
     uw = tt.vector('uw')
     uw.tag.test_value = np.concatenate([inarray.tag.test_value,
-                                        inarray.tag.test_value])
+                                        inarray.tag.test_value]).astype(np.float32)
```

Comment:
okay, just changed the type. Do not have machine to test at home but 95% confident it works.